src/main/java/org/openmrs/standalone/ApplicationController.java

```diff
@@ -243,17 +253,20 @@ public void run() {
 			if (applyDatabaseChange == DatabaseMode.USE_INITIALIZATION_WIZARD) {
 				deleteActiveDatabase();
 				StandaloneUtil.resetConnectionPassword();
-				StandaloneUtil.startupDatabaseToCreateDefaultUser();
+				StandaloneUtil.startupDatabaseToCreateDefaultUser(mySqlPort);
+				System.out.println("Database mode using wizard: " + applyDatabaseChange );
```

Comment:
Do you still need the System.out.println statements?

Reply:
Added these to be shown in the logs